king_phisher/catalog.py

```diff
@@ -344,6 +348,52 @@ def from_url(cls, url, keys=None, encoding='utf-8'):
 		keys.verify_dict(data, signature_encoding='base64')
 		return cls(data, keys=keys)
 
+class CatalogManager(object):
+	"""
+	Base manager for handling multiple Catalogs
+	"""
+	def __init__(self, url_catalog=None):
+		self.catalogs = {}
+
+		if url_catalog:
+			self.add_catalog_url(url_catalog)
+
+	def catalog_ids(self):
+		"""
+		The key names of the catalogs in the manager
+		:return: the catalogs ids in CatalogManager
+		:rtype: list
+		"""
+		return self.catalogs.keys()
+
+	def get_repos(self, catalog_id):
```

Comment:
I'd prefer for the public method names to be consistent with the attribute names e.g. `get_repository(...`